Week-3/CSS-ADVANCED/q2.html

```diff
@@ -0,0 +1,175 @@
+oatmeal_cookies.html
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <style>
+
+        body {
+        margin: 0;
+        font-family: Helvetica, sans-serif;
+        }
+
+        header {
```

Comment:
Consider adding` align-items: center;` to vertically center the items within the header for better visual alignment.